ingestion/src/metadata/orm_profiler/orm/converter.py

```diff
@@ -192,3 +199,21 @@ def get_orm_schema(table: Table, metadata: OpenMetadata) -> str:
     )
 
     return str(schema.name.__root__)
+
+
+def get_orm_database(table: Table, metadata: OpenMetadata) -> str:
+    """get database name from database service
+
+    Args:
+        table (Table): table entity
+        metadata (OpenMetadata): metadata connection to OM server instance
+
+    Returns:
+        str
+    """
+
+    database: databaseService.DatabaseService = metadata.get_by_id(
```

Comment:
Is the typing here correct? This might need to be a `Database` and not a DatabaseService

Reply:
yes good catch.